src/AWS.Deploy.CLI/Commands/TypeHints/DockerfilePathCommand.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using AWS.Deploy.Common;
+using AWS.Deploy.Common.IO;
+using AWS.Deploy.Common.Recipes;
+using AWS.Deploy.Common.TypeHintData;
+
+namespace AWS.Deploy.CLI.Commands.TypeHints
+{
+    /// <summary>
+    /// Typehint that lets the user specify a Dockerfile outside of the default location.
+    /// </summary>
+    public class DockerfilePathCommand : ITypeHintCommand
+    {
+        private readonly IConsoleUtilities _consoleUtilities;
+        private readonly IFileManager _fileManager;
+
+        public DockerfilePathCommand(IConsoleUtilities consoleUtilities, IFileManager fileManager)
+        {
+            _consoleUtilities = consoleUtilities;
+            _fileManager = fileManager;
+        }
+
+        /// <summary>
+        /// Not implemented, alternative Dockerfile paths are not suggested to the user
+        /// </summary>
+        /// <returns>Empty list</returns>
+        public Task<List<TypeHintResource>?> GetResources(Recommendation recommendation, OptionSettingItem optionSetting) => Task.FromResult<List<TypeHintResource>?>(null);
+
+        /// <summary>
+        /// Prompts the user to enter an alternative Dockerfile path
+        /// </summary>
+        public Task<object> Execute(Recommendation recommendation, OptionSettingItem optionSetting)
+        {
+            var settingValue = _consoleUtilities
+               .AskUserForValue(
+                   string.Empty,
+                   recommendation.GetOptionSettingValue<string>(optionSetting),
+                   allowEmpty: true,
+                   resetValue: recommendation.GetOptionSettingDefaultValue<string>(optionSetting) ?? "",
+                   validators: dockerfilePath => ValidateDockerfilePath(dockerfilePath));
+
+            recommendation.DeploymentBundle.DockerfilePath = settingValue;
+            return Task.FromResult<object>(settingValue);
+        }
+
+        /// <summary>
+        /// This method will be invoked to set the Docker execution directory in the deployment bundle
```

Comment:
...invoked to set the **Docker file path**.

Reply:
This whole file was dropped on the move to a generic `FIlePathCommand`